connectivity/suite.go

```diff
@@ -66,7 +68,18 @@ func Run(ctx context.Context, ct *check.ConnectivityTest) error {
 
 	// Network Performance Test
 	if ct.Params().Perf {
-		ct.NewTest("network-perf").WithScenarios(
+		netperfTest := ct.NewTest("network-perf")
+		for _, perfPolicyFile := range ct.Params().PerfPolicyFiles {
+			policyYAML, err := os.ReadFile(perfPolicyFile)
```

Comment:
Could we apply an generic policy, if the user doesn't specify a policy?

Reply:
Sure! What kind of policy do you have in mind?

Reply:
We could add a policy which allows the server port for netperf.